pkg/oc/bootstrap/docker/openshift/admin.go

```diff
@@ -92,15 +92,18 @@ func (h *Helper) InstallRegistry(dockerClient dockerhelper.Interface, ocImage st
 	}
 
 	svc, err := kubeClient.Core().Services(DefaultNamespace).Get(SvcDockerRegistry, metav1.GetOptions{})
-	if err == nil {
```

Comment:
@deads2k seems like a logical error to me that was here forever ?!

Reply:
I mean, the `RegistryServiceIP` was only set when the above returned error which in that case we delete and recreate with correct IP? does not make sense...

Reply:
also more I'm looking at this, I this we should nuke this code path and rather fix `oc adm registry` if we really need to override the ClusterIP ?